examples/complete/README.md

```diff
@@ -0,0 +1,51 @@
+# atlas-basic - complete
+
+This example shows how the atlas-basic module can be used to create:
+
+- A MongoDB Atlas project.
+- A MongoDB Atlas cluster provisioned with AWS provider for the project.
+- Several IP addresses and CIDR blocks for the project.
+- Several database users for the project.
+
+## Usage
+
+First, you need to set the following variables: 
+
+- `org_id`: ID of Atlas organization
+- `public_key`: Atlas public key
+- `private_key`: Atlas  private key
+
+Then, to run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | ~> 1.0 |
+
+## Providers
```

Comment:
thes tables seem a bit "overkilled" but I suppose it's the normal way to document modules

Reply:
Agree. But if Ithey are used I would expect a version for the `mongodbatlas`? (L36)

Reply:
I copied them from the AWS README raw ^^ https://raw.githubusercontent.com/terraform-aws-modules/terraform-aws-vpc/master/examples/complete/README.md but if it doesnt look good i can remove them and just put it in text